Source/UrlRequest_Windows.cpp

```diff
@@ -212,7 +228,8 @@ namespace UrlLib
         }
 
         Foundation::Uri m_uri{nullptr};
-        Storage::Streams::IBuffer m_responseBuffer{};
+        Storage::Streams::IBuffer m_httpResponseBuffer{};
+        std::vector<char> m_fileResponseBuffer;
```

Comment:
Is it possible to use `std::byte` instead?

Reply:
`m_fileResponseBuffer.assign(std::istreambuf_iterator<char>(file), std::istreambuf_iterator<char>());` doesn't work with `std::byte`, so I used `char` instead. Maybe there's a different way to assign `m_fileResponseBuffer` with `std::byte`?

Reply:
I did a quick search and I see https://stackoverflow.com/a/48965683. Maybe [this function](https://en.cppreference.com/w/cpp/io/basic_istream/read) is better?